kobo/apps/openrosa/libs/renderers/renderers.py

```diff
@@ -58,7 +58,7 @@ def filter_renderers(self, renderers, format):
 
 class MediaFileRenderer(BaseRenderer):
     media_type = '*/*'
-    format = None
+    format = 'TODO'
```

Comment:
This one needs to be fixed. Please add a FIXME comment and let's not forget about it.